bash/drive-utils.bash

```diff
@@ -1,5 +1,38 @@
 #!/usr/bin/env bash
 
+###################################################
+# A simple wrapper to check tempfile for access token and make authorized oauth requests to drive api
+###################################################
+_api_request() {
+    . "${TMPFILE}_ACCESS_TOKEN"
```

Comment:
Codacy found an issue: [Can't follow non-constant source. Use a directive to specify location.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6363604)